pkg/pipelines/helper.go

```diff
@@ -4,49 +4,49 @@ import (
 	"bytes"
 	"fmt"
 	"strings"
-	"time"
 
 	"github.com/getgauge-contrib/gauge-go/testsuit"
 	"github.com/openshift-pipelines/release-tests/pkg/clients"
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline"
-	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
+	v1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+// Unused
 // collectMatchingEvents collects list of events under 5 seconds that match
 // 1. matchKinds which is a map of Kind of Object with name of objects
 // 2. reason which is the expected reason of event
-func collectMatchingEvents(c *clients.Clients, namespace string, kinds map[string][]string, reason string) ([]*corev1.Event, error) {
-	var events []*corev1.Event
-
-	watchEvents, err := c.KubeClient.Kube.CoreV1().Events(namespace).Watch(c.Ctx, metav1.ListOptions{})
-	// close watchEvents channel
-	defer watchEvents.Stop()
-	if err != nil {
-		return events, err
-	}
-
-	// create timer to not wait for events longer than 5 seconds
-	timer := time.NewTimer(5 * time.Second)
-
-	for {
-		select {
-		case wevent := <-watchEvents.ResultChan():
-			event := wevent.Object.(*corev1.Event)
-			if val, ok := kinds[event.InvolvedObject.Kind]; ok {
-				for _, expectedName := range val {
-					if event.InvolvedObject.Name == expectedName && event.Reason == reason {
-						events = append(events, event)
-					}
-				}
-			}
-		case <-timer.C:
-			return events, nil
-		}
-	}
-}
+// func collectMatchingEvents(c *clients.Clients, namespace string, kinds map[string][]string, reason string) ([]*corev1.Event, error) {
+// 	var events []*corev1.Event
+
+// 	watchEvents, err := c.KubeClient.Kube.CoreV1().Events(namespace).Watch(c.Ctx, metav1.ListOptions{})
+// 	// close watchEvents channel
+// 	defer watchEvents.Stop()
+// 	if err != nil {
+// 		return events, err
+// 	}
+
+// 	// create timer to not wait for events longer than 5 seconds
+// 	timer := time.NewTimer(5 * time.Second)
+
+// 	for {
+// 		select {
+// 		case wevent := <-watchEvents.ResultChan():
+// 			event := wevent.Object.(*corev1.Event)
+// 			if val, ok := kinds[event.InvolvedObject.Kind]; ok {
+// 				for _, expectedName := range val {
+// 					if event.InvolvedObject.Name == expectedName && event.Reason == reason {
+// 						events = append(events, event)
+// 					}
+// 				}
+// 			}
+// 		case <-timer.C:
+// 			return events, nil
+// 		}
+// 	}
+// }
```

Comment:
let's remove unused code completely, it will stay in git history if we need it in the future

Reply:
Done